index.js

```diff
@@ -148,6 +150,8 @@ function randomCase(input) {
 function convert(input, shouldLetterBeUpperCase) {
   return input.replace(/./g, (str, i) => (shouldLetterBeUpperCase(str, i) ? str.toUpperCase() : str.toLowerCase()));
 }
-if (typeof window === "undefined") {
+mockingcase = mOcKiNgCaSe;
+
+try {
   module.exports = mOcKiNgCaSe;
-}
+} catch (e) {} // eslint-disable-line no-empty
```

Comment:
Maybe we can log some warning:

```suggestion
} catch (e) {
  console.warn('mOcKiNgCaSe is available in browser through functions: \n\n- mockingcase\n- mOcKiNgCaSe');
}
```

Reply:
Yeah but I feel that'd be unwanted, as it'd just pollute the console. If people were already bothering to pull it into the browser, they'd be able to figure out the name of the function. And other libraries like JQuery don't console.log anything when they're pulled in anyway.